script.js

```diff
@@ -0,0 +1,231 @@
+
+class Player {
+	constructor(name, symbol) {
+		this.name = name,
+		this.symbol = symbol
+	}
+}
+
+const Board = ( () => {
+	const boardArray = [null, null, null, null, null, null, null, null, null];
+	const winCombinations = [
+					[0,1,2],
+					[3,4,5],
+					[6,7,8],
+					[0,3,6],
+					[1,4,7],
+					[2,5,8],
+					[0,4,8],
+					[6,4,2]
+					];
+    
+	function render() {
+		const boardDiv = document.getElementById('board');
+		const cells = document.getElementsByClassName("sqr");
+		if (cells.length === 0){
+			for (let i=0; i<boardArray.length; i++) {
+			let div = document.createElement("div");
+			div.classList.add("sqr");
+			div.id = `${i}`;
+			boardDiv.appendChild(div);
+		   } 	
+		}else{
+			for (let i=0; i< cells.length; i++){
+				cells[i].innerText = "";
+				cells[i].style.removeProperty('background-color');
+			}
+		}
+		
+		
+	}
+
+	function checkWinner(symb){
```

Comment:
A simpler way to do this is for each player object to have an array consisting of all the choices the player already has made. And then we can use `every` and `some` array methods to check whether any `winCombinations` inner array elements would be contained within that player's set of moves.